### ISSUE TITLE
How does --hidden work?

### ISSUE THREAD

**Issue by ChrisWren** [node-inspector/node-inspector#206]
Can't figure out how to use the --hidden flag. I tried the following commands with no luck:

``` bash
node-inspector --hidden=[index.js]
```

``` bash
node-inspector --hidden=["index.js"]
```

``` bash
node-inspector --hidden=index.js
```

Has anyone ever configured this flag? 

Thanks.
